### PR TITLE
Add LinqPad snippets and change project target to class library

### PR DESCRIPTION
**Pull request by asindarov**

### CODE REVIEW

**Comment by asindarov** [comment]
In order to run LinqPad snippets you need to download LinqPad. This is the link to download linqPad https://www.linqpad.net/. Just play and enjoy with the snippets

**Review by Ukrainiangirl — DISMISSED** [review]

**Review by vldmrklnk — DISMISSED** [review]

File: samples/linqpad-samples/face-detection-samples/detect-face-from-image-base64.linq [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <Namespace>Exadel.Compreface.DTOs.FaceDetectionDTOs.FaceDetectionBase64</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.FaceVerificationDTOs.FaceVerification</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.RecognitionDTOs.VerifyFacesFromImage</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.RecognitionDTOs.VerifyFacesFromImageWithBase64</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.SubjectDTOs.AddSubject</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.SubjectDTOs.DeleteSubject</Namespace>
+  <Namespace>Exadel.Compreface.DTOs.SubjectDTOs.RenameSubject</Namespace>
+</Query>
+
+var faceRecognitionClient = new FaceDetectionClient("PASTE YOUR API KEY", "http://localhost", "8000");
```
Comment (vldmrklnk) [inline samples/linqpad-samples/face-detection-samples/detect-face-from-image-base64.linq]: Wrong name of variable.

**Comment by vldmrklnk** [comment]
So, here is one problem. I created examples which don't display result in console. Because I used to fiddler where all response's data is being displayed. So, some examples need functionality to being displayed in console.

File: samples/linqpad-samples/face-detection-samples/detect-face-from-image-base64.linq (hunk 1 above)
Comment (asindarov) [reply samples/linqpad-samples/face-detection-samples/detect-face-from-image-base64.linq]: Yeah, it is my bad, I renamed correctly

File: Exadel.Compreface.AcceptenceTests/Services/FaceDetectionServiceTest.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ public async Task FaceDetectionBase64Async_TakesRequestModel_ReturnsProperRespon
             var response = await _faceDetectionService.FaceDetectionBase64Async(_faceDetectionBase64Request);
 
             // Assert
-            Assert.IsType<FaceDetectionBase64Response>(response);
+            Assert.IsType<FaceDetectionResponse>(response);
```
Comment (asindarov) [inline Exadel.Compreface.AcceptenceTests/Services/FaceDetectionServiceTest.cs:91]: the reason for this change is that `FaceDetectionBase64Response ` class is not satisfied the contract with api response. The declaration of FaceDetectionBase64Response:

```
public class FaceDetectionBase64Response : BaseResult
    {
        public PluginVersions PluginVersions { get; set; }
    }
```
but the compreface postman docs shows that the response will be identical with fields `FaceDetectionResponse` so we don't need duplicate classes. It seems like a convention base64 methods will return same response fields with regular endpoints so for detection we don't need extra base64 response

File: samples/linqpad-samples/face-verification-samples/verify-image-base64.linq [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    SourceImageWithBase64 = "", // source image's base64 value,
+    TargetImageWithBase64 = "", // target image's base64 value,
+    Status = true,
+};
+
+var faceVerificationResponse = await faceVerificationClient.FaceVerificationService.VerifyBase64ImageAsync(faceVerificationWith64Request);
+
+foreach(var result in faceVerificationResponse.Result)
+{
+	// observe result object
```
Comment (asindarov) [inline samples/linqpad-samples/face-verification-samples/verify-image-base64.linq:42]: I leave this comment to ensure that user can play with this object on his/her own

**Review by Ukrainiangirl — APPROVED** [review]

**Review by vldmrklnk — APPROVED** [review]

**Review by Evgeniy-Sinicin — APPROVED** [review]

**Pull request merged**